app/views/OverviewPageView.scala.html

```diff
@@ -93,12 +115,14 @@ <h2 class="app-task-list__section">
 }
 
 @taskWithActionOrCaption(heading: String, action: String, caption: String, index: Int, incomeSources: IncomeSourcesModel) = {
+
+    @if(appConfig.giftAidReleased){
     <li>
         <h2 class="app-task-list__section">
             <span class="app-task-list__section-number">@index.</span> @heading
         </h2>
         <li class="app-task-list__items">
-            @if(incomeSources.interest.isDefined || incomeSources.dividends.isDefined) {
+            @if(incomeSources.interest.isDefined || incomeSources.dividends.isDefined || incomeSources.giftAid.isDefined) {
```

Comment:
"incomeSources.giftAid.isDefined" could be changed to "(incomeSources.giftAid.isDefined && appConfig.giftAidReleased)" which would remove the need for the new if else